<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>panel</class>
 <widget class="QWidget" name="panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Stereo Calibration</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>优化选项</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>300</width>
      <height>284</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="nintrinsics">
         <property name="text">
          <string>NINTRINSICS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="intrinsicsGuess">
         <property name="text">
          <string>USE_INTRINSICS_GUESS</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="fixAspectRatio">
         <property name="text">
          <string>FIX_ASPECT_RATIO</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixPrincipalPoint">
         <property name="text">
          <string>FIX_PRINCIPAL_POINT</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="fixFocalLength">
         <property name="text">
          <string>FIX_FOCAL_LENGTH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="sameFocalLength">
         <property name="text">
          <string>SAME_FOCAL_LENGTH</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="fixK1">
         <property name="text">
          <string>FIX_K1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixK2">
         <property name="text">
          <string>FIX_K2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixK3">
         <property name="text">
          <string>FIX_K3</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixK4">
         <property name="text">
          <string>FIX_K4</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="fixK5">
         <property name="text">
          <string>FIX_K5</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixK6">
         <property name="text">
          <string>FIX_K6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixS1S2S3S4">
         <property name="text">
          <string>FIX_S1_S2_S3_S4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QCheckBox" name="rationalModel">
         <property name="text">
          <string>RATIONAL_MODEL</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="thinPrismModel">
         <property name="text">
          <string>THIN_PRISM_MODEL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="tiltedModel">
         <property name="text">
          <string>TILTED_MODEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="fixTAUXTAUY">
         <property name="text">
          <string>FIX_TAUX_TAUY</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QCheckBox" name="useQR">
         <property name="text">
          <string>USE_QR</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useLU">
         <property name="text">
          <string>USE_LU</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QCheckBox" name="fixTangentDist">
         <property name="text">
          <string>FIX_TANGENT_DIST</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="zeroTangentDist">
         <property name="text">
          <string>ZERO_TANGENT_DIST</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QCheckBox" name="fixIntrinsics">
         <property name="text">
          <string>FIX_INTRINSICS</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="zeroDisparity">
         <property name="text">
          <string>ZERO_DISPARITY</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QCheckBox" name="extrinsicsGuess">
         <property name="text">
          <string>USE_EXTRINSICS_GUESS</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>终止条件：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>迭代次数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterNum">
         <property name="maximum">
          <number>65535</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>精度：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="eps">
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="precision">
         <property name="currentIndex">
          <number>5</number>
         </property>
         <item>
          <property name="text">
           <string>e-1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e-2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e-3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e-4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e-5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e-6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>e-7</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>标定</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>文件:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>图样格式：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="setRows">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="setCols">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>行</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>列</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="setInterval">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>间隔：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detectCorners">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>检测角点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>计算参数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>角点数：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cornerNums">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>图片对：</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgNums">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存标定结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCorners">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存角点图片</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="toGray">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>首先转为灰度图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetParams">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>重置参数</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
